manager/state/raft/raft.go

```diff
@@ -1092,6 +1151,11 @@ func (n *Node) reportNewAddress(ctx context.Context, id uint64) error {
 	if err != nil {
 		return err
 	}
+	if oldAddr == "" {
```

Comment:
It's quite weird. `newPeer` calls `grpc.Dial` which is supposed to check if address is empty :/